db/migrate/20220628222858_add_user_account_uuid_to_in_progress_forms.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserAccountUuidToInProgressForms < ActiveRecord::Migration[6.1]
+
+  def change
+    safety_assured do
+      add_reference :in_progress_forms, :user_account, type: :uuid, foreign_key: true, null: true, index: true
```

Comment:
You'll have to break out any /db and schema changes to a separate PR

Reply:
👍 Rebased on top of the merged database migration, so db/schema is out of this PR now

Reply:
Thanks! GHA appears to be a bit wonky (having issues today) could you re-run the CI pipeline?